creator-node/src/diskManager.js

```diff
@@ -82,6 +82,29 @@ class DiskManager {
     return path.join(parentDirPath, cid)
   }
 
+  /**
+   * Construct the legacy path to a file or directory given a CID
+   */
+  static computeLegacyFilePath(cid) {
+    if (!this.isValidCID(cid)) {
+      throw new Error(`[computeLegacyFilePath] [CID=${cid}] Invalid CID.`)
+    }
+    return path.join(this.getConfigStoragePath(), 'files', cid)
+  }
+
+  /**
+   * Boolean function to check if arg is a valid CID
+   */
+  static isValidCID(cid) {
+    try {
```

Comment:
does this ever fail?

Reply:
not sure, just copying code from the above `computeFilePath` and didnt want to change it as part of this change
i assume the reason it was added is bc there was some error case
a @dmanjunath question for sure

Reply:
just looked at the docs, and it looks like if you pass in some bad values, this line will fail https://github.com/multiformats/js-cid/blob/9d5574deee7ab327998e63b616bda57495fd9dfe/src/index.js#L174. 
this looks good then. resolving

Reply:
the `new CID()` line will fail if the input is not good. the `isCID(cid)` call will not

Reply:
sweet thanks for confirming